modules/rosa-sts-creating-cluster.adoc

```diff
@@ -64,11 +64,12 @@ I: To determine when your cluster is Ready, run `rosa describe cluster <cluster_
 $ rosa create cluster --interactive
 ----
 
-. Enter the following command to check the status of your cluster and retrieve your cluster ID. During cluster creation, the `State` field from the output will transition from `pending` to `installing`, and finally to `ready`.
+. Enter the following command to check the status of your cluster and retrieve your cluster ID. During cluster creation,
+the `State` field from the output will transition from `pending` to `installing`, and finally to `ready`:
 +
 [source,terminal]
 ----
-$ rosa describe cluster --cluster=<cluster_name>
+$ rosa describe cluster --cluster=${name}
```

Comment:
Are the curly braces required as part of the input?

Reply:
The usage of `${name}` was intended to follow the same usage from the `rosa create cluster` command in https://deploy-preview-34246--osdocs.netlify.app/openshift-rosa/latest/rosa_getting_started/rosa-sts-creating-cluster.html#clipboard-0